ntc_templates/templates/index

```diff
@@ -401,6 +401,7 @@ cisco_nxos_show_hostname.textfsm, .*, cisco_nxos, sh[[ow]] hostn[[ame]]
 cisco_nxos_show_hsrp_all.textfsm, .*, cisco_nxos, sh[[ow]] hsrp all
 cisco_nxos_show_ip_route.textfsm, .*, cisco_nxos, sh[[ow]] ip route
 cisco_nxos_show_feature.textfsm, .*, cisco_nxos, sh[[ow]] feat[[ure]]
+cisco_nxos_show_nve_vni_interface_nve_1_detail.textfsm, .*, cisco_nxos, sh[[ow]] nv[[e]] vn[[i]] int[[erface]] nv[[e]] 1 det[[ail]]
```

Comment:
This does not comply with the index ordering requirements: https://ntc-templates.readthedocs.io/en/latest/dev/dev_parser/#index-file

Reply:
@jmcgill298 Ordering has been corrected, thanks!